WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -243,7 +243,7 @@
           </Viewbox>
           <DockPanel Grid.Column="0" Grid.Row="0" VerticalAlignment="Center">
             <TextBlock Classes="text"
-                       Text="Wasabi provides privacy by default, it protects your financial information through coinjoins, client-side block filtering and communication over the Tor anonymity network.&#10;&#10;Neither the public nor the developers can breach your privacy." />
+                       Text="Wasabi provides privacy by default, it protects your important financial information.&#10;&#10;Neither the public nor the developers can breach your privacy.&#10;&#10;To protect your privacy, you pay a fee to the Bitcoin network and the Wasabi backend." />
```

Comment:
here I add a mention of the fees paid, this should maybe be highlighted in a dedicated screen, or with some graphics?

it's arguably the most important information for new users.